loopy/kernel/array.py

```diff
@@ -892,6 +892,9 @@ def __init__(self, name, dtype=None, shape=None, dim_tags=None, offset=0,
                 tags=tags,
                 **kwargs)
 
+    def __hash__(self):
+        return super().__hash__()
```

Comment:
Could you add a link to the Python docs that explains why this needs to be redefined in this way? (This'll help avoid folks stumbling on this in the future and removing it again.)

Reply:
Done.